examples/remix/app/routes/buckets/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { ActionFunction, Form, useLoaderData } from "remix";
+import { s3Client } from "../../../backend/s3Client.server";
+import { createBucket, listBuckets } from "../../../backend/buckets.server";
```

Comment:
Let's move `s3Client.server` and `buckets.server` to `app/services/s3` and create and `index.ts` file there that exports `*` from those files:
```
export * from "./s3Client.server";
export * from "./buckets.server";
```
So, here we can use:
```suggestion
import { s3Client, createBucket, listBuckets } from "~/services/s3";
```
Remix recommends having everything inside the app folder. It also makes it easier to import files using `~`.